blockchain/utxocache.go

```diff
@@ -142,9 +145,13 @@ func (entry *UtxoEntry) memoryUsage() uint64 {
 	}
 
 	// This value is calculated by running the following on a 64-bit system:
-	//   unsafe.Sizeof(UtxoEntry{})
+	//   unsafe.Sizeof(UtxoEntry{}) before adding tokenData
 	baseEntrySize := uint64(40)
 
+	// This value is calculated by running the following on a 64-bit system:
+	//   unsafe.Sizeof(TokenData{}) assuming commitment of length 40
+	baseEntrySize += uint64(88)
```

Comment:
These magic integers should probably be constants with the comment located by the const.